### PR TITLE
Release fixes #102358

### LINKED ISSUE #102358: SCM: View becomes empty, with errors in dev tools
Issue Type: <b>Bug</b>

Since the new design of the source control has been introduced (old one was way better) VSC has a problem loading all gitlab repos after pushing once. The source control is then completely empty and if i try to sort new, an error pops up "template could not be loaded"

VS Code version: Code 1.47.0 (d5e9aa0227e057a60c82568bf31c04730dc15dcd, 2020-07-09T08:02:06.629Z)
OS version: Windows_NT x64 10.0.18363

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i9-9900K CPU @ 3.60GHz (16 x 3600)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off_ok<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.94GB (7.34GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (11)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-eslint|dba|2.1.6
vscode-html-css|ecm|0.2.3
code-runner|for|0.11.0
pythonsnippets|frh|1.0.2
git-graph|mhu|1.24.0
prettify-json|moh|0.0.3
vscode-docker|ms-|1.3.1
python|ms-|2020.6.91350
markdown-preview-enhanced|shd|0.5.12
vscode-counter|uct|2.0.0
vscodeintellicode|Vis|1.2.9

</details>

### PR DESCRIPTION
**Pull request by joaomoreno**
This PR fixes #102358 for 1.47

### CODE REVIEW

File: src/vs/workbench/contrib/scm/browser/scmViewPane.ts [hunk 1]
```diff
@@ -316,19 +316,19 @@ class InputRenderer implements ICompressibleTreeRenderer<ISCMInput, FuzzyScore,
...
-			initialRender();
-		}
+			if (contentHeight !== InputRenderer.DEFAULT_HEIGHT) {
+				onDidChangeContentHeight();
+			}
+		};
+
+		// Setup height change listener on next tick
+		const timeout = setTimeout(startListeningContentHeightChange, 0);
+		disposables.add({ dispose: () => clearTimeout(timeout) });
```
Comment (bpasero) [inline src/vs/workbench/contrib/scm/browser/scmViewPane.ts:331]: We have a `toDisposable` helper for this

**Review by bpasero — APPROVED** [review]
Not being able to reproduce the original issue, a review is harder. But I tested the multiline input field and it still seems to be working fine.

Only semantic change I noticed:
previously `onDidChangeContentHeight` was always called if `contentHeight === InputRenderer.DEFAULT_HEIGHT` - this is not the case anymore

File: src/vs/workbench/contrib/scm/browser/scmViewPane.ts (hunk 1 above)
Comment (joaomoreno) [reply src/vs/workbench/contrib/scm/browser/scmViewPane.ts:331]: This somehow survived the huge refactoring. We actually have a `disposableTimeout` helper which is even better. Will change it in `master`!

**Comment by joaomoreno** [comment]
> Only semantic change I noticed:
> previously `onDidChangeContentHeight` was always called if `contentHeight === InputRenderer.DEFAULT_HEIGHT` - this is not the case anymore

I actually through it was safe to remove it, but your comment made me retest it in another way and I was able to repro another issue because of this. So now I always call it.

File: src/vs/workbench/contrib/scm/browser/scmViewPane.ts (hunk 1 above)
Comment (joaomoreno) [reply src/vs/workbench/contrib/scm/browser/scmViewPane.ts:331]: https://github.com/microsoft/vscode/commit/5278689345deb4758421f3da3d87ce21d7e43697

**Review by bpasero — APPROVED** [review]
Yeah better now. Maybe could get a comment as to why this needs a `setTimeout`

**Pull request merged**